builtin/provisioners/chef/ssh_provisioner.go

```diff
@@ -0,0 +1,74 @@
+package chef
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform/communicator"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const (
+	installURL = "https://www.chef.io/chef/install.sh"
```

Comment:
Hey @vmorris 

Thank you for your comment. This Pull Request has been merged for over a year, and it's difficult for the Terraform team to triage closed and merged issues. Could you please open a new issue for them to track. Thanks! 😄